game-core/src/main/java/games/strategy/triplea/ResourceLoader.java

```diff
@@ -75,8 +73,10 @@ public static ResourceLoader getGameEngineAssetLoader() {
 
   /** Returns a resource loader that will find assets in a map directory. */
   public static ResourceLoader getMapResourceLoader(final String mapName) {
-    final List<String> dirs = getPaths(mapName);
-    if (mapName != null && dirs.isEmpty()) {
+    Preconditions.checkNotNull(mapName);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'Preconditions.checkNotNull' due to existing static import 'com.google.common.base.Preconditions.checkNotNull'](https://app.codacy.com/gh/triplea-game/triplea/pullRequest?prid=6139462)